configuration/types.go

```diff
@@ -41,6 +41,7 @@ const (
 	BotCreated                   WebhookAction = "bot_created"
 	BotUpdated                   WebhookAction = "bot_updated"
 	BotDeleted                   WebhookAction = "bot_deleted"
+	ListGroupsProperties         WebhookAction = "list_groups_properties"
```

Comment:
There isn't any webhook named `list_groups_properties`.

Reply:
Removed any indication list_groups_properties is a hook.